curvefs/src/mds/metaserverclient/metaserver_client.cpp

```diff
@@ -221,7 +253,7 @@ FSStatusCode MetaserverClient::CreateCopySet(uint32_t poolId,
         copyset->set_poolid(poolId);
         copyset->set_copysetid(id);
         for (auto item : addrs) {
-            copyset->add_peers()->set_address(item + std::to_string(0));
+            copyset->add_peers()->set_address(item + ":0");
```

Comment:
hard code ":0" is obscure, it's better to add another `BuildPeerId` for this case

Reply:
> hard code ":0" is obscure, it's better to add another `BuildPeerId` for this case

fixed